var/spack/repos/builtin/packages/grackle/package.py

```diff
@@ -42,6 +42,10 @@ class Grackle(Package):
     version('2.2', 'ec49ed1db5a42db21f478285150c2ba3')
     version('2.0.1', 'a9624ad13a60c592c1a0a4ea8e1ae86d')
 
+    variant('float',
+            default=False,
+            description='Build with float')
```

Comment:
Just OCD kicking in, but can you please put this variant on a single line (if it is less than 80 chars)?

Reply:
Done